lib/vmpooler/api/reroute.rb

```diff
@@ -3,85 +3,69 @@
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called post /vm? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm")
       end
 
       post '/vm/:template/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called post /vm/:template/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:template]}")
       end
 
       get '/vm/:hostname/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called /vm/:hostname/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}")
       end
 
       delete '/vm/:hostname/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called delete /vm/:hostname/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}")
       end
 
       put '/vm/:hostname/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called put /vm/:hostname/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}")
       end
 
       post '/vm/:hostname/snapshot/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called post /vm/:hostname/snapshot/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}/snapshot")
       end
 
       post '/vm/:hostname/snapshot/:snapshot/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called post /vm/:hostname/snapshot/:snapshot/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}/snapshot/#{params[:snapshot]}")
       end
 
       put '/vm/:hostname/disk/:size/?' do
-        puts "DEPRECATION WARNING a client (#{request.user_agent}) called put /vm/:hostname/disk/:size/? and got redirected to api_version=1, this behavior will change in the next major version, please modify the client to use v2 in advance"
         call env.merge('PATH_INFO' => "/api/v#{api_version}/vm/#{params[:hostname]}/disk/#{params[:size]}")
```

Comment:
Can we just kill this off and simply use the versioned endpoints as part of the breaking changes already being introduced?